frameworks/g-cloud-9/search_mapping.json

```diff
@@ -1,4 +1,27 @@
 {
+  "settings": {
+    "index": {
+      "number_of_shards": 1
```

Comment:
5 shards is the Elasticsearch default, so I suspect it was just that.

We may need to increase replication so that each node gets a copy of the primary shard. I'm not sure how the number of replicas is assigned at the moment given we have different numbers of nodes in preview/staging vs prod. I will have to look into that.